envpool/mujoco/dmc/cheetah.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2022 Garena Online Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ENVPOOL_MUJOCO_DMC_CHEETAH_H_
+#define ENVPOOL_MUJOCO_DMC_CHEETAH_H_
+
+#include <algorithm>
+#include <limits>
+#include <memory>
+#include <random>
+#include <string>
+
+#include "envpool/core/async_envpool.h"
+#include "envpool/core/env.h"
+#include "envpool/mujoco/dmc/mujoco_env.h"
+#include "envpool/mujoco/dmc/utils.h"
+
+namespace mujoco_dmc {
+
+std::string GetCheetahXML(const std::string& base_path,
+                          const std::string& task_name) {
+  return GetFileContent(base_path, "cheetah.xml");
+}
+
+class CheetahEnvFns {
+ public:
+  static decltype(auto) DefaultConfig() {
+    return MakeDict("max_episode_steps"_.Bind(1000), "frame_skip"_.Bind(4),
```

Comment:
```suggestion
    return MakeDict("max_episode_steps"_.Bind(1000), "frame_skip"_.Bind(1),
```

Reply:
you can find n_sub_step and time_limit in rl/control.py class Environment `__init__`

Reply:
Get it！ So `_n_sub_steps` means `frame_skip`, `_step_limit`  means `n_sub_step`.

Reply:
y